packages/core/src/Stores/client-store.js

```diff
@@ -314,6 +324,11 @@ export default class ClientStore extends BaseStore {
             : eu_excluded_regex.test(this.residence);
     }
 
+    @computed
+    get is_uk() {
+        return this.residence === 'gb';
+    }
+
```

Comment:
Will we be making more extensive use of this getter later on?

Reply:
Oh yeah. it has its own set of rules, however we can rely on the if equal statement throughout.